examples/geometry/image_profile.ipynb

```diff
@@ -0,0 +1,128 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x23f3ac88588>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
```

Comment:
The reduceImageProfile function seems to be the most important part of this example. This could be improved by adding notebook annotations that explain what the function is doing.